certified-connectors/Repfabric Lead Loader/apiDefinition.swagger.json

```diff
@@ -0,0 +1,357 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Lead Loader",
+    "description": "This Connector helps automate the process of creating a \"Lead\" for Repfabric's backend system. It takes data from a csv and invokes multiple calls to the Repfabric system that can create or get the items needed to automate the Lead creation process.",
+    "contact": {
+      "name": "Phiquest Team",
+      "url": "https://phiquest.com/",
+      "email": "andre@phiquest.io"
+    },
+    "version": "1.0"
+  },
+  "host": "xe8g8pkprc.execute-api.us-east-2.amazonaws.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": [],
+  "paths": {
+    "/contact": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Status Number after finish Lambda Function"
+                },
+                "data": {
+                  "type": "string",
+                  "description": "Message describing what has been created in the Repfabric system"
+                },
+                "probability": {
+                  "type": "string",
+                  "description": "Probability that the Principal used matches the one in the Repfabric system"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Upload leads and create an opportunity",
+        "description": "Upload leads and create an opportunity",
+        "operationId": "UploadLeadsV3",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "company": {
+                  "type": "string",
+                  "description": "company"
```

Comment:
@andreroberts1119 please make a bit more detailed descriptions for properties, you can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions) and to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).